packages/auth/test/utils_test.js

```diff
@@ -438,11 +438,11 @@ function testMatchDomain_ipAddressDomain() {
 
 function testMatchDomain_caseInsensitiveMatch() {
   assertTrue(fireauth.util.matchDomain('localhost', 'localhost', 'http'));
-  assertTrue(fireauth.util.matchDomain('domain.com', 'DOMAIN.COM', 'http'));
+  assertTrue(fireauth.util.matchDomain('domain.com', 'DOMAIN_NAME.COM', 'http'));
```

Comment:
I think you accidentally replaced the text here.

Reply:
Thanks! Good catch. I reverted it.
I am quite surprised that this did not break the build.